core/tests/protractor_utils/general.js

```diff
@@ -256,6 +256,15 @@ var navigateToTopicsAndSkillsDashboardPage = async function() {
   await waitFor.pageToFullyLoad();
 };
 
+var navigateToBlogDashboardPage = async function() {
```

Comment:
Can you put this in `BlogDashboardPage.js` instead?

Reply:
Done.